src/sections/Partners/index.js

```diff
@@ -246,6 +232,17 @@ const Partner = () => {
                         </div>
                     </Col>
                 </Row>
+                <Row>
+                    <Col xs={12} sm={4} lg={4}>
```

Comment:
Why these extra <Col> tags?

Reply:
Yes, please. Do that using CSS only, this is not a good way to do it, I think. Also, please try to avoid inline CSS as much as possible 🙂

Reply:
Fixed. Also removed the inline css that was already there and put it in the stylesheet.

Reply:
Why both justify-content and text-align?
Also, can you please fix the DCO issue.

Reply:
What is the DCO issue?